docs/modules/kafka/pages/index.adoc

```diff
@@ -2,45 +2,66 @@
+Follow the xref:kafka:getting_started/index.adoc[] which will guide you through installing The Stackable Kafka and
+ZooKeeper Operators, setting up ZooKeeper and Kafka and testing your Kafka using kcat.
 
 == Resources
 
-The _KafkaCluster_ custom resource contains your Kafka cluster configuration. It defines a single `broker` xref:concepts:roles-and-role-groups.adoc[role].
+The _KafkaCluster_ custom resource contains your Kafka cluster configuration. It defines a single `broker`
+xref:concepts:roles-and-role-groups.adoc[role].
 
 image::kafka_overview.drawio.svg[A diagram depicting the Kubernetes resources created by the operator.]
 
-For every xref:concepts:roles-and-role-groups.adoc#_role_groups[role group] in the `broker` role the Operator creates a StatefulSet. Multiple Services are created - one at role level, one per role group as well as one for every individual Pod - to allow access to the entire Kafka cluster, parts of it or just individual brokers.
+For every xref:concepts:roles-and-role-groups.adoc#_role_groups[role group] in the `broker` role the Operator creates a
+StatefulSet. Multiple Services are created - one at role level, one per role group as well as one for every individual
+Pod - to allow access to the entire Kafka cluster, parts of it or just individual brokers.
 
-For every StatefulSet (role group) a ConfigMap is deployed containing a `log4j.properties` file for xref:usage-guide/logging.adoc[logging] configuration and a `server.properties` file containing the whole Kafka configuration which is derived from the KafkaCluster resource.
+For every StatefulSet (role group) a ConfigMap is deployed containing a `log4j.properties` file for
+xref:usage-guide/logging.adoc[logging] configuration and a `server.properties` file containing the whole Kafka
+configuration which is derived from the KafkaCluster resource.
 
-The Operator creates a xref:concepts:service_discovery.adoc[] for the whole KafkaCluster which references the Service for the whole cluster. Other operators use this ConfigMap to connect to a Kafka cluster simply by name and it can also be used by custom third party applications to find the connection endpoint.
+The Operator creates a xref:concepts:service_discovery.adoc[] for the whole KafkaCluster which references the Service
+for the whole cluster. Other operators use this ConfigMap to connect to a Kafka cluster simply by name and it can also
+be used by custom third party applications to find the connection endpoint.
 
 == Dependencies
 
-Kafka requires xref:zookeeper:index.adoc[Apache ZooKeeper] for coordination purposes (it will not be needed in the future as it will be replaced with a https://cwiki.apache.org/confluence/display/KAFKA/KIP-500%3A+Replace+ZooKeeper+with+a+Self-Managed+Metadata+Quorum[built-in solution]).
+Kafka requires xref:zookeeper:index.adoc[Apache ZooKeeper] for coordination purposes (it will not be needed in the
+future as it will be replaced with a {metadata-quorum}[built-in solution]).
```

Comment:
I don't think having this variable for this link is helpful, maybe just put the link here?